river/tree/hoeffding_adaptive_tree_classifier.py

```diff
@@ -60,7 +60,7 @@ class HoeffdingAdaptiveTreeClassifier(HoeffdingTreeClassifier):
         potential replacement to the current one.
     drift_detector
         The drift detector used to build the tree. If `None` then `drift.ADWIN` is used.
-    switch_delta
+    switch_signif
```

Comment:
@smastelini I would prefer full parameter name, so `switch_significance` in this case

Reply:
Done!